src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/WebRequestMethod.cs

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+#nullable enable
+
 namespace Microsoft.PowerShell.Commands
 {
```

Comment:
Please move such trivial files in another PR. This will simplify follow review.
We should annotate base types before.
Aslso I don't see WebRequestSession. It is good candidate for new separate PR.

Reply:
I worked on WebRequestSession here: https://github.com/PowerShell/PowerShell/pull/19291